tests/unit/handlers/test__helpers.py

```diff
@@ -106,61 +161,122 @@ def test_valid_context_header(self):
 
         middleware = request.RequestMiddleware(None)
         middleware.process_request(django_request)
-        trace_id = self._call_fut()
+        http_request, trace_id = self._call_fut()
 
         self.assertEqual(trace_id, expected_trace_id)
+        self.assertEqual(http_request["request_method"], "GET")
+        self.assertEqual(set(http_request.keys()), set(_HTTP_REQUEST_FIELDS))
+        for field in _HTTP_REQUEST_FIELDS:
+            self.assertTrue(field in http_request)
 
+    def test_http_data(self):
```

Comment:
please name the test so it's more concrete in name what is being tested. What exactly in http data?